src/actions/format_actions.ts

```diff
@@ -130,6 +152,75 @@ export const formatNumberDuration: ActionSpec = {
   isActive: (env) => isFormatSelected(env, "hhhh:mm:ss"),
 };
 
+export const moreFormats: ActionSpec = {
+  name: _t("More formats"),
+  execute: (env) => env.openSidePanel("MoreFormats", {}),
+};
+
+const EXAMPLE_DATE = toNumber("2023/09/22 10:00:00 PM", DEFAULT_LOCALE);
+
+const fullDateTimeFormat = "dddd d mmmm yyyy hh:mm:ss a";
+export const formatNumberFullDateTime: ActionSpec = {
+  name: _t("Full date time"),
+  description: formatValue(EXAMPLE_DATE, { locale: DEFAULT_LOCALE, format: fullDateTimeFormat }),
+  execute: (env) => setFormatter(env, fullDateTimeFormat),
+  isActive: (env) => isFormatSelected(env, fullDateTimeFormat),
+};
```

Comment:
Idea: create a `crateDateFormatActionSpec(name, format)` helper, all of these actions are straight up copy/pasted. Probably a good idea ?